src/libsystemd/sd-journal/journal-file.h

```diff
@@ -296,3 +292,21 @@ static inline uint8_t COMPRESSION_TO_MASK(Compression c) {
                 return 0;
         }
 }
+
+static inline uint8_t COMPRESSION_TO_FLAG(Compression c) {
```

Comment:
hmm, so we now have COMPRESSION_TO_MASK() and COMPRESSION_TO_FLAG(). That's confusing!

Maybe rename COMPRESSION_TO_MASK() to COMPRESSION_TO_OBJECT_FLAG()? and then name this new helper call here COMPRESSION_TO_INCOMPATIBLE_HEADER_FLAG()?

also the incompatible header flags field is a uint32_t in `struct Header`, hence the 
COMPRESSION_TO_INCOMPATIBLE_HEADER_FLAG() helper should also return that